manifests/f/FirebirdProject/Firebird/4/4.0.3.2975/FirebirdProject.Firebird.4.installer.yaml

```
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: FirebirdProject.Firebird.4
+PackageVersion: 4.0.3.2975
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/FirebirdSQL/firebird/releases/download/v4.0.3/Firebird-4.0.3.2975-0-x64.exe
+  InstallerSha256: DB2091AC5E1C232366B37946ECC3B7DF751BC98669791389C49CD137E9C8EBC1
+  AppsAndFeaturesEntries:
+  - DisplayName: Firebird 4.0.3.2975 (x64)
+    Publisher: Firebird Project
+    DisplayVersion: 4.0.3.2975
+    ProductCode: FBDBServer_4_0_x64_is1

```

```suggestion
  - DisplayName: Firebird 4.0.3.2975 (x64)
    Publisher: Firebird Project
    ProductCode: FBDBServer_4_0_x64_is1
```
DisplayVersion should not be used when it's equal to PackageVersion. See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147